lsp-bridge.el

```diff
@@ -1491,7 +1483,7 @@ So we build this macro to restore postion after code format."
    ))
 
 (defun lsp-bridge--not-in-org-table ()
-  (not (and (boundp 'org-at-table-p)
+  (not (and (fboundp 'org-at-table-p)
```

Comment:
Since `org-at-table-p` is a function, not a variable, this code did not work as expected.